src/System.Net.Requests/src/System/Net/FtpWebRequest.cs

```diff
@@ -0,0 +1,2544 @@
+// ------------------------------------------------------------------------------
+// <copyright file="FtpWebRequest.cs" company="Microsoft">
+//     Copyright (c) Microsoft Corporation.  All rights reserved.
+// </copyright>
+// ------------------------------------------------------------------------------
+//
+
+namespace System.Net {
+    using System.Collections;
+    using System.IO;
+    using System.Text;
+    using System.Net.Sockets;
+    using System.Net.Cache;
+    using System.Threading;
+    using System.Security;
+    using System.Security.Cryptography.X509Certificates ;
+    using System.Security.Authentication;
+    using System.Globalization;
+    using System.Diagnostics.Tracing;
+    using System.Threading.Tasks;
```

Comment:
Not sure if the CodeFormatter will take care of this, but according to the [coding style](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md):

> 6) Namespace imports should be specified at the top of the file, outside of namespace declarations and should be sorted alphabetically.